packages/design-picker/src/available-designs-config.json

```diff
@@ -424,6 +424,21 @@
 			"is_premium": false,
 			"is_fse": true,
 			"features": []
+		},
+		{
+			"title": "Start with an empty page",
```

Comment:
Noting that we won't be able to translate this text if we keep this data in JSON format

cc @StefanNieuwenhuis and @razvanpapadopol

Reply:
Yes, probably we're going to change https://github.com/Automattic/wp-calypso/blob/trunk/packages/design-picker/src/components/index.tsx#L85 to something like:
```
<span className="design-picker__option-name">{ design.slug === 'blank-canvas' ? __('"Start with an empty page", __i18n_text_domain__ ) : design.title }</span>
```
Of course, the translation function call should be outside ternary 😅 

I guess we can add this to the list of customizations that `DesignPicker` should support for the black canvas design option alongside showing it always first in the list.

Reply:
Added to the A/Cs of #52016